### PR TITLE
Add maxHeight prop to DropdownButton

### PR DESCRIPTION
**Pull request by sradevski**
Connects-to: #678
Change-type: minor
Signed-off-by: Stevche Radevski <stevche@balena.io>

### CODE REVIEW

File: src/components/DropDownButton.tsx [hunk 1]
```diff
@@ -184,6 +185,11 @@ class DropDownButton extends React.Component<
 						alignRight={alignRight}
 						onClick={this.toggle}
 						minWidth={`${this.state.minWidth}px`}
+						maxHeight={
+							isNumber(this.props.maxHeight)
```
Comment (LucianBuzzo) [inline src/components/DropDownButton.tsx]: You can use the `px()` function from `utils.ts` here https://github.com/balena-io-modules/rendition/blob/master/src/utils.ts#L40

File: src/stories/DropDownButton.js [hunk 2]
```diff
@@ -36,6 +36,18 @@ storiesOf('Core/DropDownButton', module)
             <div>Item</div>
             <div>Item</div>
           </DropDownButton>
+          <DropDownButton
+            mx={2}
+            maxHeight={120}
+            label={<div>Fixed Height</div>}
```
Comment (LucianBuzzo) [inline src/stories/DropDownButton.js]: Fixed height and max height are subtly different, perhaps changing this to `Max Height` is better?

File: src/components/DropDownButton.tsx (hunk 1 above)
Comment (sradevski) [reply src/components/DropDownButton.tsx]: Oh cool, thanks!

File: src/stories/DropDownButton.js (hunk 2 above)
Comment (sradevski) [reply src/stories/DropDownButton.js]: True, I will change it.

File: src/components/DropDownButton.tsx (hunk 1 above)
Comment (sradevski) [reply src/components/DropDownButton.tsx]: @LucianBuzzo Done, have a look at it when you find the time.

**Pull request merged**